config/traits.yml

```diff
@@ -2080,8 +2080,8 @@ traits:
       type: numeric
       units: MPa
       label: Leaf turgor loss point
-      allowed_values_min: .01
-      allowed_values_max: 100
+      allowed_values_min: -100
```

Comment:
Numbers look ok, but I can't really tell without knowing the data.

Reply:
These are the traits where some people report "inverted" values (as in, positive water potentials, when they are actually negative) and the units had been "neg_MPa", indicating the value in AusTraits was the negative of the "real number". We decided to switch everything to the true units, but for several traits I didn't swap around the allowed_values.

Reply:
Yes, I gathered. I more meant that I can't tell if the indicated upper and lower values are reasonable or not. So